utils/build.ps1

```diff
@@ -843,7 +900,7 @@ function Build-Compilers($Arch, [switch]$Test = $false) {
       }
     }
 
-    $env:Path = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\runtime-development\usr\bin;${env:Path}"
+    $env:Path = (Get-PinnedToolchainRuntime) + ";${env:Path}"
```

Comment:
I think that I prefer the Swift style of string interpolation instead.

```suggestion
    $env:Path = "$(Get-PinnedToolchainRuntime);${env:Path}"
```

Reply:
Done